.github/workflows/helm-release.yaml

```diff
@@ -17,16 +17,17 @@ on:
         required: false
         default: ".github/lint-config.yaml"
         type: string
-      python-version:
```

Comment:
You could still forward it to `bakdata/ci-templates/actions/bump-version@v1.5`

Reply:
our bump-version action no longer supports passing a specific Python version in the latest release